docs/en/advanced/Client-Logging.md

```diff
@@ -6,14 +6,12 @@ group: Advanced
 priority: 1
 ---
 
-* Table of Contents
-{:toc}
-
-## Introduction
-
 This page summarizes how to find client-specific logs and modify logging levels in order to debug
 client-side issues with your Alluxio installation.
 
+* Table of Contents
```

Comment:
I saw some TOCs are in the beginning of the article. shall we keep it consistent?